app/interactors/retrieve_token_stats.rb

```diff
@@ -22,7 +22,7 @@ def raw_body
   end
 
   def stats_url
-    "https://dashboard.entreprise.api.gouv.fr/api/watchdoge/stats/jwt_usage/#{context.token_id}"
+    "https://dashboard.entreprise.api.gouv.fr/api/watchdoge/stats/token_usage/#{context.token_id}"
```

Comment:
```suggestion
    "https://dashboard.entreprise.api.gouv.fr/api/watchdoge/stats/jwt_usage/#{context.token_id}"
```

Il s'agit de watchdoge